project/src/main/java/com/google/sps/servlets/EditUserServlet.java

```diff
@@ -46,12 +53,13 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
                     userEmail,
                     (String)entity.getProperty("password"),
                     (String)entity.getProperty("birthdate"),
-                    Long.parseLong(String.valueOf(entity.getProperty("studentId"))),
+                    (long) entity.getProperty("studentId"),
                     (String)entity.getProperty("sex"),
                     (String)entity.getProperty("school"),
                     (String)entity.getProperty("phone"),
                     (String)entity.getProperty("metric"),
-                    Boolean.parseBoolean(String.valueOf(entity.getProperty("admin"))));
+                    (boolean) entity.getProperty("admin"),
```

Comment:
just FYI, usually, we use is_xx if the variable is boolean type.

Reply:
Thank you, I will definitely use that in the future.